Gems/EMotionFX/Code/Source/Editor/SkeletonModel.h

```diff
@@ -99,6 +100,9 @@ namespace EMotionFX
         static const char* s_simulatedJointIconPath;
         static const char* s_simulatedColliderIconPath;
 
+        static bool indexIsRootNode(const QModelIndex& idx);
```

Comment:
suggestion: As this isn't overriding a Qt function we should likely make it and the neighboring call `UpperCamelCase` to comply with the coding standards